### ISSUE TITLE
Adjusting hyperparamters from a previous training session?

### ISSUE THREAD

**Issue by jlanis** [Unity-Technologies/ml-agents#591]
I was wondering if it's considered "best practice" to adjust hyperparamters from an earlier training session compared to starting a new training session from the beginning. For example, a common problem is that the algorithm starts off well, but then gets to a point where it gets "stuck" and the cumulative reward pretty much levels off (even though it's possible to keep increasing). Let's say I want to adjust a certain hyperparamter to see if it performs better. After adjusting the parameter, would it be recommended to load the existing training model, or start from the beginning? Or does it not make much difference?

**Comment by vincentpierre** [comment]
You can change some hyperparameters but not all of them. You can change the learning rate, the number of max steps, beta, batch and buffer size but you cannot change parameters that define the structure of the network (number of hidden layers, hidden units, memories).
I would recommend starting from scratch to avoid potential problems on the structure of the network.

**Comment by xiaomaogy** [comment]
@jlanis Thanks for reaching out to us. Hopefully you were able to resolve your issue.  We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.
